server/src/model/init.ts

```diff
@@ -35,6 +36,7 @@ const addInitialVoteSchema = async () => {
   schema.addPointer("board", "Board", {required: true});
   schema.addPointer("note", "Note", {required: true});
   schema.addPointer("user", "_User", {required: true});
+  schema.addNumber("votingIteration", {required: true, defaultValue: 0});
```

Comment:
I think I wouldn't specify a defaultValue here because when creating a vote without specifying the `votingIteration` it's wrong to assume that the board is in the 0th iteration.

Reply:
If we specify a default value, we don't need to handle the undefined case separately (incremented by one each voting iteration).